docs/development_environment.md

```diff
@@ -78,6 +78,26 @@ Install the requirements with a provided script named `setup`.
 script/setup
 ```
 
+:::tip
+The first time a WSL distribution is started, and the default user account created, the Windows drives will be mounted with all files owned by `root:root`:
+```
+user@DESKTOP:/mnt/c/Users/user$ mount | grep mnt
+C:\ on /mnt/c type drvfs (rw,noatime,uid=0,gid=0,case=off)
+```
+
+This will cause the `setup` script to fail if the local repository is on a Windows drive. To recover, the WSL subsystem must be restarted. The easiest method is to simply restart the computer.
```

Comment:
```suggestion
This will cause the `setup` script to fail if the local repository is on a Windows drive. To recover, WSL  must be restarted. The easiest method is to simply restart the computer.
```

WSL = Windows Subsystem for Linux, so previous was Windows Subsystem for Linux subsystem